documentation/ansible-playbook/modules/ansible-playbook-create-bonded-interfaces-using-script.adoc

```diff
@@ -0,0 +1,128 @@
+[id="ansible-playbook-create-bonded-interfaces-using-script.adoc"]
+
+= Create bonded interfaces
+
+If you need to create a bonded provisioning/baremetal interface, the ifcfg files
+can be placed under the "fake" roots for the masters and workers under the
+`roles/installer/files/customize_filesystem/{master,worker}` directories as shown in the above section 4.7
```

Comment:
Is this targetted for `4.7` or later or is it only tested on `4.4.5` as mentioned?

Reply:
This is only tested on 4.4.5 and upgrades to 4.5.x. This has not been tested on 4.6 or 4.7